solidjs-tailwind/src/components/UserDropdown/user-dropdown.classNames.js

```diff
@@ -1,7 +1,10 @@
 export const dropdown = 'relative top-1 inline-block text-left z-30 relative';
-export const dropdownBtn = 'inline-flex items-center text-gray-200 fill-current';
-export const avatarContainer = 'w-8 h-8 rounded-full overflow-hidden flex items-center justify-center bg-gray-200';
+export const dropdownBtn =
```

Comment:
I mentioned this in other PR reviews.
Can we switch to using CSS module just the way Nextjs implemented it?

This styling was used in Remix because Remix does not support CSS module.

Reply:
can we create a ticket to fix this, this only joined this PR because i did prettier at some point

Reply:
Okay sure